lib/src/sdk.dart

```diff
@@ -174,4 +179,178 @@ class BeansMerchantSdk {
       throw Exception('Unknown class');
     }
   }
+
+  /// Creates a sub-account for the company
+  ///
+  /// [stellarAccountId] The Stellar account ID for the sub-account
+  /// [name] The name of the sub-account in different languages as a map where
+  /// the key is the language code (e.g., 'en', 'vi') and the value is the name in that language
+  Future<CreateCompanyAccountResponse> createCompanyAccount(
+    String stellarAccountId,
+    Map<String, String> name,
+  ) async {
+    final response = await httpClient.post(
+      Uri.parse('$apiBaseUrl/companies/me/account'),
+      headers: {
+        'Content-Type': 'application/json',
+        'X-Beans-Company-Api-Key': apiKey,
+      },
+      body: jsonEncode({
+        'stellarAccountId': stellarAccountId,
+        'name': name,
+      }),
+    );
+
+    if (response.statusCode == 201) {
+      return CreateCompanyAccountResponse.fromJson(jsonDecode(response.body));
+    } else {
+      throw ApiException(
+        response.statusCode,
+        response,
+        'Failed to create company account',
+      );
+    }
+  }
+
+  /// Uploads an avatar for a company sub-account
+  ///
+  /// [companyId] The ID of the company or 'me' for the current company
+  /// [stellarAccountId] The Stellar account ID of the sub-account
+  /// [imageBytes] The image data as bytes
+  /// [mimeType] The MIME type of the image (e.g., 'image/jpeg', 'image/png')
+  Future<CompanyAccount> uploadCompanyAccountAvatar(
+    String companyId,
+    String stellarAccountId,
+    Uint8List imageBytes,
```

Comment:
Can we not best accept a file path? Then we know the extension and can determine a mime type right? Seems like this would make it more easy for the end user. Or both of course.